### DISCUSSION TITLE
How are setup modules identified?

### DISCUSSION THREAD

**Discussion by tve** [Moddable-OpenSource/moddable#1021] [Q&A] [answered]
I read https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/base/setup.md but I can't see anything telling me how I identify a new setup module to the build system. Please point me to the docs for that or add it to the doc:-), thanks!

**Comment by phoddie** [comment, marked as answer]
Set-up modules are those modules with a module secifier that begins with `setup/`.
